iceoryx_meta/CMakeLists.txt

```diff
@@ -17,6 +17,7 @@ file (STRINGS "../VERSION" iceoryx_VERSION)
 project(iceoryx_meta VERSION ${iceoryx_VERSION})
 
 option(TOML_CONFIG "activates or deactivates TOML support - without TOML RouDi will not be build" ON)
+option(ONE_TO_MANY "Restricts communication to 1:n pattern" OFF)
```

Comment:
Maybe ONE_TO_MANY_ONLY, to make more clear that it is a restriction. When I read MANY here I get the feeling that it extends and not restricts

Reply:
Done